clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -323,7 +323,11 @@ DPCT_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::list<DPCPPExtensions>), NoDPCPPE
              DPCT_OPT_ENUM("enqueued_barriers",
                                      int(DPCPPExtensions::Ext_EnqueueBarrier),
                                      "Enqueued barriers extension.",
-                                     false)
+                                     false),
+             DPCT_OPT_ENUM("free_mem_size",
```

Comment:
https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_device_info.md 
the extension name is:  sycl_ext_intel_device_info, how about use "device_info" in the new option value?

Reply:
Could you also test if SYCL CPU device can run the function, if not, in helper function we need update the imp.

Reply:
#1. done
#2: on SYCL CPU device, it will give a runtime msg below from the helper function get_sycl_mem_info(), so no need to update it.
```
erminate called after throwing an instance of 'std::runtime_error'
  what():  ext_intel_free_memory is not supported.
Aborted (core dumped)
```